.github/workflows/release.yml

```diff
@@ -117,7 +117,7 @@ jobs:
             build-profile: production
             package: frequency-runtime
             runtime-dir: runtime/frequency
-            built-wasm-file-name-prefix: frequency_rococo_runtime
+            built-wasm-file-name-prefix: frequency_runtime
```

Comment:
this is the file generated by srtool for runtime specified so it will never be anything but frequency_runtime.x

Its just you will rename that to either rococo/mainnet if  am correct

check the error here https://github.com/LibertyDSNP/frequency/actions/runs/3600119600/jobs/6064429748

Reply:
Example
https://github.com/LibertyDSNP/frequency/actions/runs/3600119600/jobs/6064429748#step:6:1528

Given we have only one runtime srtool will spit out same named files for both of rococo and mainnet as it takes "name" of the file == {runtime_name}.compress.wasm.x.x.etc